wgpu-core/src/device/mod.rs

```diff
@@ -2774,16 +2774,11 @@ impl<A: HalApi> Device<A> {
                             },
                         )?;
                     }
-                    Some(&None) => {
-                        return Err(
-                            pipeline::CreateRenderPipelineError::InvalidFragmentOutputLocation(*i),
-                        );
-                    }
                     _ => {
-                        return Err(pipeline::CreateRenderPipelineError::ColorState(
-                            *i as u8,
-                            pipeline::ColorStateError::Missing,
-                        ));
+                        log::warn!(
```

Comment:
I don't think we should warn on this - it's going to be really noisy if people intentionally are doing this.

Reply:
Is it okay to change it to log::info!()? which is helpful for users when setting up `RUST_LOG=info` to debug the program.

Reply:
Yeah, let's do info. We should probably re-evaluate logging levels at some point.